### PR TITLE
change pretty-calldata labels from decimal ints to hex values

### PR DESCRIPTION
**Pull request by DJViau**
This PR changes the labels on pretty-calldata from decimal integers to hex labels, like one finds in the debugger.

For example, 

```
 [0]:  0000000000000000000000000000000000000000000000000000000000000020
 [1]:  0000000000000000000000000000000000000000000000000000000000000002
 [2]:  0000000000000000000000000000000000000000000000000000000000000040
 [3]:  0000000000000000000000000000000000000000000000000000000000000140
 [4]:  000000000000000000000000da5a5adc64c8013d334a0da9e711b364af7a4c2d
 [5]:  0000000000000000000000000000000000000000000000000000000000000000
 [6]:  0000000000000000000000000000000000000000000000000000000000000000
 [7]:  0000000000000000000000000000000000000000000000000000000000000080
 [8]:  0000000000000000000000000000000000000000000000000000000000000044
 [9]:  40c10f19000000000000000000000000e05fcc23807536bee418f142d19fa0d2
 [10]: 1bb0cff700000000000000000000000000000000000000000000000000000000
 ...
```

becomes

```
 [000]: 0000000000000000000000000000000000000000000000000000000000000020
 [020]: 0000000000000000000000000000000000000000000000000000000000000002
 [040]: 0000000000000000000000000000000000000000000000000000000000000040
 [060]: 0000000000000000000000000000000000000000000000000000000000000140
 [080]: 000000000000000000000000da5a5adc64c8013d334a0da9e711b364af7a4c2d
 [0a0]: 0000000000000000000000000000000000000000000000000000000000000000
 [0c0]: 0000000000000000000000000000000000000000000000000000000000000000
 [0e0]: 0000000000000000000000000000000000000000000000000000000000000080
 [100]: 0000000000000000000000000000000000000000000000000000000000000044
 [120]: 40c10f19000000000000000000000000e05fcc23807536bee418f142d19fa0d2
 [140]: 1bb0cff700000000000000000000000000000000000000000000000000000000
 ...
```

## Motivation

The integer labels require mental energy to convert to hex, which is how my team typically thinks of these.  Converting to hex saves time and mental energy.

## Solution

I updated the display method for `PossibleSigs`.

NOTE: This is my first time writing rust and my first time making this type of contribution.  I read the contribution guidelines and attempted to follow their instructions, but I found some of the cargo checks hard to parse.  I skipped opening an issue since the change is tiny and cosmetic.  Please let me know what I need to do to properly comply with the letter and spirit of the contribution rules.

EDIT: This change also makes it easier to look up offsets.

### CODE REVIEW

**Comment by jameswenzel** [comment]
imo much easier for parsing out calldata offsets in calls with structs or dynamic parameters

**Comment by DJViau** [comment]
re: the label @mattsse assigned: this changes the behavior of `cast pretty-calldata` so that it behaves the same way the debugger does, so this should probably get the C-cast label in addition or instead.

**Review by mattsse — APPROVED** [review]
sgtm
